src/consensus/pbft/libbyz/LedgerWriter.cpp

```diff
@@ -42,73 +44,17 @@ void LedgerWriter::write_prepare(
       (uint8_t*)&(p.second),
       sizeof(Prepared_cert::PrePrepareProof));
   }
-
-  ledger->put_entry(entry);
```

Comment:
Should this be removed? It seems that now, `write_prepare()` has no effect?

Reply:
Yes all things manually writing to the ledger will be removed in next PR